<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AiCodeReview</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="uploadButton">
      <property name="text">
       <string>파일 업로드</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="filePathTextEdit">
      <property name="placeholderText">
       <string>업로드된 파일 경로가 여기에 표시됩니다...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reviewButton">
      <property name="text">
       <string>GPT API로 리뷰 요청</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="reviewOutput">
      <property name="placeholderText">
       <string>리뷰 결과가 여기에 표시됩니다...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>

